backend/src/index.js

```diff
@@ -104,10 +99,12 @@ io.on( 'connection', socket => {
 			return;
```

Comment:
There's a logic leak here. Because in such case we still need to update the DB and broadcast updated users (as your state might have been changed).

Reply:
Good point